### PR TITLE
Added Support For TLevels

### PR DESCRIPTION
**Pull request by Gurnaik-Randhawa**
### This release contains 
Support for TLevel API calls
Unit Tests for TLevels

### CODE REVIEW

File: DFC.FindACourseClient.IntegrationTests/CourseSearchServiceTests.cs [hunk 1]
```diff
@@ -65,12 +67,38 @@ public async Task CourseSearch()
...
             };
 
             var courseSearchService = new CourseSearchApiService(findACourseClient, auditService, mapper);
             var searchResponse = await courseSearchService.SearchCoursesAsync(courseSearchRequest).ConfigureAwait(false);
+
+            searchResponse.Should().NotBeNull();
+        }
+
+        [Fact]
+        public async Task CopositeCourseSearch()
```
Comment (keenesolutions) [inline DFC.FindACourseClient.IntegrationTests/CourseSearchServiceTests.cs:81]: Typo in method name.

File: DFC.FindACourseClient.IntegrationTests/CourseSearchServiceTests.cs [hunk 2]
```diff
@@ -65,12 +67,38 @@ public async Task CourseSearch()
...
+            };
+
+            var courseSearchService = new CourseSearchApiService(findACourseClient, auditService, mapper);
+            var searchResponse = await courseSearchService.SearchCoursesAsync(courseSearchRequest).ConfigureAwait(false);
+
+            searchResponse.Should().NotBeNull();
+        }
+
+        [Fact]
+        public async Task CopositeTLevelSDetails()
```
Comment (keenesolutions) [inline DFC.FindACourseClient.IntegrationTests/CourseSearchServiceTests.cs:96]: Typo in method name.

File: DFC.FindACourseClient.UnitTests/FindACourseClientTLevelTests.cs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+            // Act
+            var result = await findACourseClient.TLevelGetAsync(tLevelId.ToString()).ConfigureAwait(false);
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.Equal(expectedResponse.Qualification.TLevelName, result.Qualification.TLevelName);
+            Assert.Equal(expectedResponse.TLevelId, result.TLevelId);
+
+            httpResponse.Dispose();
```
Comment (keenesolutions) [inline DFC.FindACourseClient.UnitTests/FindACourseClientTLevelTests.cs:64]: All of these implement IDisposable and could therefore be using statements.

**Review by keenesolutions — CHANGES_REQUESTED** [review]
Just a couple of typos.

File: DFC.FindACourseClient.IntegrationTests/CourseSearchServiceTests.cs (hunk 2 above)
Comment (Gurnaik-Randhawa) [reply DFC.FindACourseClient.IntegrationTests/CourseSearchServiceTests.cs]: Fixed

File: DFC.FindACourseClient.IntegrationTests/CourseSearchServiceTests.cs (hunk 1 above)
Comment (Gurnaik-Randhawa) [reply DFC.FindACourseClient.IntegrationTests/CourseSearchServiceTests.cs]: Fixed

**Review by keenesolutions — APPROVED** [review]
Looks good.

**Pull request merged**